src/server/sched_func.c

```diff
@@ -335,5 +335,21 @@ set_sched_default(pbs_sched* psched) {
 		psched->sch_attr[(int) SCHED_ATR_sched_cycle_len].at_flags = ATR_VFLAG_DEFLT
 				| ATR_VFLAG_SET | ATR_VFLAG_MODCACHE;
 	}
+	if ((psched->sch_attr[(int) SCHED_ATR_sched_state].at_flags & ATR_VFLAG_SET)
+			==0) {
```

Comment:
I really dislike line breaks like this one.  You only save 6 characters at the cost of readability.